src/mavsdk_server/src/grpc_server.cpp

```diff
@@ -142,6 +142,10 @@ int GrpcServer::run()
     builder.RegisterService(&_tune_service);
 #endif
 
+#ifdef BUILD_WITH_PROTO_REFLECTION
```

Comment:
Wait, does this have an effect? I don't think cmake will propagate it from `-DBUILD_WITH_PROTO_REFLECTION` to C++. You have to explicitly set it from the CMakeLists, right? Or does this work? :thinking:

Reply:
You caught me there ;). commited with the fix.